internal/version.go

```diff
@@ -5,8 +5,9 @@ import "github.com/Masterminds/semver/v3"
 var (
 	Branch = "main"
 	// {x-release-please-start-version}
-	Version = "0.13.0"
+	realVersion = "0.13.0"
```

Comment:
`realVersion` should only be modified by release automation and hold valid semantic versions

Reply:
^ I think we should add this comment in the `version.go` file